rust/agama-cli/src/main.rs

```diff
@@ -26,6 +27,15 @@ use std::{
     time::Duration,
 };
 
+/// Agama's CLI global options
+#[derive(Args)]
+struct GlobalOpts {
+    #[clap(long)]
+    /// uri pointing to agama's remote api. If not provided, default https://localhost/api is
```

Comment:
Currently the option is required:
```console
 # agama config show
error: the following required arguments were not provided:
  --uri <URI>

Usage: agama --uri <URI> <COMMAND>

For more information, try '--help'.
```

Reply:
reason is that it should be just ` pub uri: Option<String>,` so it does not need to be provided.

Reply:
yes ... during testing i used it as mandatory ... and it is already fixed in one of upcoming commits.